### ISSUE TITLE
Bug in Augmentation code 

### ISSUE THREAD

**Issue by Sandeep418** [feiyuhuahuo/Yolact_minimal#29]
I am  plotting the bounding boxes after the augmentation Its not matching with ground truth boxes .
The red color box is the Ground truth box
white color box is getting from after augmentation
![image](https://user-images.githubusercontent.com/40691260/120154068-16ee2000-c20d-11eb-8b11-efa7d3856dd6.png)
![image](https://user-images.githubusercontent.com/40691260/120154148-2cfbe080-c20d-11eb-87ac-311b269e78d4.png)
![image](https://user-images.githubusercontent.com/40691260/120154212-43a23780-c20d-11eb-9658-f03addeecb52.png)
![image](https://user-images.githubusercontent.com/40691260/120154278-5583da80-c20d-11eb-84b0-152291ec8cc1.png)
![image](https://user-images.githubusercontent.com/40691260/120154364-6df3f500-c20d-11eb-9775-b110a30f348b.png)

![image](https://user-images.githubusercontent.com/40691260/120154456-89f79680-c20d-11eb-9d55-3129693454e6.png)
![image](https://user-images.githubusercontent.com/40691260/120155157-2de14200-c20e-11eb-892e-a2c0b9d61aaa.png)

**Comment by feiyuhuahuo** [comment]
That's normal. Some data augmentation functions changed the object position like `random_crop`. Thus the label coordinates should also be changed.

**Comment by Sandeep418** [comment]
check here The bounding box is shifted but the object is not shifted
![image](https://user-images.githubusercontent.com/40691260/120178952-a86a8b80-c227-11eb-8468-b26dbf5cdaa4.png)
![image](https://user-images.githubusercontent.com/40691260/120179327-0bf4b900-c228-11eb-976d-053afcf91353.png)
![Uploading image.png…]()
![Uploading image.png…]()

**Comment by feiyuhuahuo** [comment]
You can set the `num_workers` =0 in the training data_loader, then uncomment this line to visualize the annotation information. That's the image feed to the network directly. You may make something wrong.
https://github.com/feiyuhuahuo/Yolact_minimal/blob/4c3e50dde56c8a83c411c6d5d4f657b497356f3f/utils/augmentations.py#L247
![image](https://user-images.githubusercontent.com/32631344/120254376-46a83100-c2bc-11eb-9dc6-30d78378d5d7.png)

**Comment by Sandeep418** [comment]
@feiyuhuahuo Thank you for your response . 
Solved the issue by transposing the image after augmentation 
` boxes = np.hstack((boxes, np.expand_dims(custom_labels1, axis=1)))
                        img=img.transpose(1,2,0)
                        return img, boxes, masks`
